src/textual_summary/generic_table_row.jsx

```diff
@@ -22,7 +22,7 @@ const renderMultivalue = function renderMultivalue(values, onClick) {
 };
 
 const renderValue = function renderValue(value, onClick) {
```

Comment:
Just a small nitpick. You don't have to write `const functionName = function functionName(...)`, you can replace it with `const functionName = (...) => {}`. The function definition after the `=` is useless. I have noticed that you are using this very often.

Reply:
Or you can write just `function renderValue(value, onClick) {` which is probably the best, if it's not just simple function with one liner as return otherwise go ahead and use `const functionName = (...) => ({some: object})`